templates.dist/form/applicantform.php

```diff
@@ -45,16 +45,20 @@
 foreach ( $pForm->getInputFields() as $input => $table ) {
```

Comment:
```suggestion
foreach ( $pForm->getInputFields() as $input => $table ) {
    if($pForm->isHiddenField($input)) {
        $hiddenValues []= renderFormField($input, $pForm);
        continue;
    }
```

Reply:
To do not add ```<br>``` I would use an hidden value array

In line 82 and 88 I would add this array
```<?php echo implode( $hiddenValues); ?>```